pkg/api/v1alpha1/verification_types.go

```diff
@@ -53,3 +57,18 @@ func (a TransmissionRiskVector) Less(i, j int) bool {
 func (a TransmissionRiskVector) Swap(i, j int) {
 	a[i], a[j] = a[j], a[i]
 }
+
+type VerificationClaims struct {
+	PHAClaims         map[string]string      `json:"phadata"`
+	TransmissionRisks TransmissionRiskVector `json:"trisk"`
+	SignedMAC         string                 `json:"tekmac"`
+	KeyVersion        string                 `json:"keyVersion"`
+	jwt.StandardClaims
```

Comment:
Pointer

Reply:
This is the way the documentation specifies
https://godoc.org/github.com/dgrijalva/jwt-go#example-NewWithClaims--CustomClaimsType

Nil pointer errors in use otherwise.